lib/google/cloud/gemserver/cli/request.rb

```diff
@@ -54,8 +54,6 @@ def initialize url = nil, proj_name = nil
           #
           # @param [String] permissions The permissions the generated key will
           # have (read, write, or both). Optional.
-          #
-          # @return [String]
```

Comment:
What do these methods return then? They clearly return something, since you're checking the return value of the health call.